src/components/lab/MeasuringArrow.js

```diff
@@ -20,24 +41,40 @@ const MeasuringArrow = ({ measuringArrowWidth, stageWidth, stageHeight }) => {
     container.style.cursor = 'default';
   };
 
-  // place arrow 10% from left and 20% from top of canvas
-  const arrowInitialXPosition = 0.1 * stageWidth;
+  // place arrow 20% from left and 20% from top of canvas
+  const arrowInitialXPosition = 0.2 * stageWidth;
   const arrowInitialYPosition = 0.2 * stageHeight;
 
+  // this is used to center text elements above the arrow in return statement below
+  const arrowCenterPoint =
+    arrowInitialXPosition +
+    (measuringArrowWidth - MEASURING_ARROW_POINTER_LENGTH) / 2;
+
   return (
     <Group draggable onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
+      <Text
+        text={String.fromCharCode(MEASURING_ARROW_SYMBOL_CHAR_CODE)}
+        ref={crossHairTextNode}
+        x={arrowCenterPoint - crossHairTextNodeWidth / 2}
+        // 'manually' position text above arrow
+        y={arrowInitialYPosition - 10 * MEASURING_ARROW_POINTER_LENGTH}
```

Comment:
Factor out the `10`?

Reply:
I think I prefer to keep this particular factor as is here, to convey that it is used in the manual positioning of the text above the arrow, rather than as some pre-defined constant and giving it a name like `MEASURING_ARROW_SYMBOL_Y_AXIS_ADJUSTMENT_FACTOR`...